MobilliumBuilders/Classes/UIScrollViewBuilder.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  UIScrollViewBuilder.swift
+//  MobilliumBuilders
+//
+//  Created by Murat Çelebi on 17.03.2021.
+//
```

Comment:
We can add copyright
```suggestion
// Copyright © 2021 Mobillium. All rights reserved.
```